### DISCUSSION TITLE
DigitalOcean App platform installation

### DISCUSSION THREAD

**Discussion by cybertaz** [directus/directus#7824] [Archive]
Hello,
Please guide me on installing directus on DO App Platform.... by following your docs I get a lot of warnings and errors when I do npm install directus in order to create the github repo.... I used the CLI instead and it installs properly locally, but now when I try to build it on DO it keeps saying that it cant connect to db.... I've put all the env including the CA ones inside the DO fields (the live build logs says it is passing the variables, but it still doesn't work) .... so I don't know what to do.... Please help me out. Thank you.

**Comment by joselcvarela** [comment]
Hi,
You can try one click install https://docs.directus.io/getting-started/installation/#one-click-installs
Or 
You can setup directus via command line. Just follow these instructions, the program will ask you for the needed information (like database address, user, etc.)
https://docs.directus.io/guides/installation/cli/

**Reply by cybertaz** [reply]
Hello, Thank you for your reply, but that is not something that I want. I would like to use the DO App Platform.

**Comment by rijkvanzanten** [comment]
What docs did you follow exactly? These https://docs.directus.io/guides/installation/digitalocean-app-platform/ ?

**Reply by cybertaz** [reply]
Hello
Yes... the App platform guide.... BUT I created the github repo after I ran the CLI install, because the manual "npm install directus" causes a lot of warnings and errors.

I'm not sure if the .env is overwritten by DO env variables set in the App Settings.

**Reply by rijkvanzanten** [reply]
> I'm not sure if the .env is overwritten by DO env variables set in the App Settings.

It'll be merged with whatever values you have in your .env. That being said, if you're using a PaaS solution like that, I would recommend omitting the .env file from your repo completely, in favor of using the env vars in App Platform
